datatree/tests/test_datatree.py

```diff
@@ -454,8 +447,41 @@ def test_roundtrip_unnamed_root(self):
         assert roundtrip.equals(dt)
 
 
-class TestBrowsing:
-    ...
+class TestDatasetView:
```

Comment:
This set of tests is insufficient in the sense that our subclass could have broken literally any method defined `xarray.Dataset`. But for now I'm just checking obvious things still work.